### PR TITLE
closes #51 - Add new variables to MH

### LINKED ISSUE #51: Add variables to mh for admiral template script and dataset
To make `admiral_admh` dataset and template script more interesting and complex within the `admiral `package we need to add variables to the sdtm dataset `mh`.  

At the moment we need the following variables could be added:
`MHOCCUR`
`MHENDTC`
`MHENRF`
`MHENTPT`

### PR DESCRIPTION
**Pull request by yangc71**
Close #51

### CODE REVIEW

**Comment by yangc71** [comment]
Hi @rossfarrugia, @bms63 I added some variables to MH based on dummy data, could you please take a look and let me know if any changes are required? Also, after running devtools::documents() the documentation for ‘admiral_ds’ ‘admiral_mh’ ‘admiral_suppds’ are deleted (because they are renamed to raw_*). I'll add those back to data.R tomorrow. Thanks!

File: inst/data_scripts/update_mh.R [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+library(labelled)
+library(admiral)
+library(metatools)
+library(admiral.test)
+
+data("raw_mh")
+
+# Convert blank to NA
+mh <- convert_blanks_to_na(raw_mh)
+set.seed(1)
```
Comment (bms63) [inline inst/data_scripts/update_mh.R:13]: Can you put a comment on why we need to set the seed?

**Review by bms63 — COMMENTED** [review]
I think this is looking good.  Ping me once you have update the `data.R` file and I can approve

File: inst/data_scripts/update_mh.R (hunk 1 above)
Comment (yangc71) [reply inst/data_scripts/update_mh.R:16]: Updated

**Comment by bms63** [comment]
@yangc71 Can you fix these merge conflicts?  Need to merge in devel to get the latest checks.

**Comment by yangc71** [comment]
> @yangc71 Can you fix these merge conflicts? Need to merge in devel to get the latest checks.

Fixed, thanks!

File: R/data.R [hunk 2]
```diff
@@ -53,13 +53,20 @@
 #' @source \url{https://github.com/cdisc-org/sdtm-adam-pilot-project/blob/master/updated-pilot-submission-package/900172/m5/datasets/cdiscpilot01/tabulations/sdtm/lb.xpt?raw=true}
 "admiral_lb"
 
-#' Medical History Dataset
+#' Medical History Dataset-updated
 #'
-#' A SDTM MH dataset from the CDISC pilot project
+#' An updated SDTM MH dataset that uses the CDISC pilot project
```
[the remark this answers is not in the document]
Comment (bms63) [reply R/data.R:58]: Do you mind changing the other ones that we updated with that phrase

File: inst/data_scripts/update_mh.R [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# Add new variables to MH
+library(dplyr)
+library(tidyselect)
+library(labelled)
```
Comment (bms63) [inline inst/data_scripts/update_mh.R:4]: Do we need labelled package?  I think add_labels() is from metatools.  Could just drop this package.

**Review by bms63 — CHANGES_REQUESTED** [review]
Can you also run styler on the code?

**Comment by bms63** [comment]
Also please put yourself as author in `DESCRIPTION` file and for `admiral_mh` like Gopi and Madhi did for ds and pc.  Another R package with Annie Yang as an author!!

File: R/data.R (hunk 2 above)
[the remark this answers is not in the document]
Comment (yangc71) [reply R/data.R]: Sure, updated.

File: inst/data_scripts/update_mh.R (hunk 3 above)
Comment (yangc71) [reply inst/data_scripts/update_mh.R]: Thanks, actually I think we only need the metatools package...

**Comment by yangc71** [comment]
> Can you also run styler on the code?

Sure, I ran styler in the latest commit. I wonder if we should put this in the PR checklist too, to run styler?

**Comment by bms63** [comment]
> > Can you also run styler on the code?
> 
> Sure, I ran styler in the latest commit. I wonder if we should put this in the PR checklist too, to run styler?

Yes!  Do you mind updating the template?

**Comment by yangc71** [comment]
> 

Thanks, I updated the PR template. Please let me know if further changes are needed.

**Review by bms63 — APPROVED** [review]
Looks great!

**Pull request merged**